static_inventories/betabarrel_cluster.yml

```diff
@@ -16,6 +16,7 @@ all:
             - name: "{{ stack_prefix }}_internal_management"
               security_group: "{{ stack_prefix }}_jumphosts"
               assign_floating_ip: true
+              assign_hostname: true
```

Comment:
```suggestion
              set_ip_to_hostname: true
```

_assigning hostname_ means something completely different. And I know we agree that variables should reflect what kind of information they carry (or do).